helm/alfresco-content-services/6.0.N_values.yaml

```diff
@@ -73,7 +73,7 @@ transformrouter:
   replicaCount: 2
   image:
     repository: quay.io/alfresco/alfresco-transform-router
-    tag: "1.0.2.1"
+    tag: "1.3.2-A2"
```

Comment:
You've updated 6.0.N for helm but not for docker compose, did you intend to do this one?

In fact, do we want to update 6.0.N at all, we normally only update latest and 6.2.N?

Reply:
reverted